Packs/EWSMailSender/Integrations/EWSMailSender/EWSMailSender.yml

```diff
@@ -82,7 +82,8 @@ script:
       description: 'A CSV list of CIDs to embed attachments within the email itself.'
       isArray: true
     description: Sends an email using EWS.
-  dockerimage: demisto/py-ews
+  dockerimage: demisto/py-ews:latest
```

Comment:
I dont think you should use demisto/py-ews:latest, I think you need to get the version of the latest, if this one is in dockerfiles.
check with @IkaDemisto

Reply:
```suggestion
  dockerimage: demisto/py-ews:2.0
```